templates/repo/settings/options.tmpl

```diff
@@ -203,8 +203,17 @@
 							{{range .PushMirrors}}
 							<tr>
 								<td class="tw-break-anywhere">{{.RemoteAddress}}</td>
-								<td>{{ctx.Locale.Tr "repo.settings.mirror_settings.direction.push"}}</td>
-								<td>{{if .LastUpdateUnix}}{{DateUtils.FullTime .LastUpdateUnix}}{{else}}{{ctx.Locale.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label" data-tooltip-content="{{.LastError}}">{{ctx.Locale.Tr "error"}}</div>{{end}}</td>
+								<td>{{ctx.Locale.Tr "repo.settings.mirror_settings.direction.push"}} ({{.Interval}})</td>
```

Comment:
Wouldn't it make more sense to include the interval already in the translated string?

Reply:
How to? See the screenshot, I do not see your point.

Reply:
And I think the current syntax is good for all languages, no need to make it more complex.

Reply:
I meant something like `Push (every %v)`, but yes, we can postpone it.

Reply:
Well, actually `Push (every %v)` is not really translatable ..... here `2h3m` is not a human language, it doesn't fit "every" in many cases. 🤣